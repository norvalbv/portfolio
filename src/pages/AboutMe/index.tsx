import React, { ReactElement } from 'react';
import CardWrapper from 'components/CardWrapper';
import StyledLink from 'components/StyledLink';
import { MY_SKILLS } from 'constants/index';

const AboutMe = (): ReactElement => {
  return (
    <CardWrapper
      title="About Me"
      className="grid w-10/12 min-w-[20rem] place-items-center text-center"
    >
      <p className="text-lg">
        Experienced Full Stack software engineer with 3 years of experience in web development. I am
        part of the core team that developed the entire&nbsp;
        <StyledLink label="Elysia" link="https://elysia.co" />
        &nbsp;&#40;a WAE subsidiary&#41; cloud platform. I would love for you to check out my
        projects and even reach out!
      </p>
      <p className="my-10 text-accent-secondary">
        My skills? They&apos;re floating around on the background. Have a look around &#58;&#41;
      </p>
      <p className="mx-auto w-6/12 min-w-[16rem] rounded border border-light-text bg-black/20 p-4 text-xs dark:border-dark-text dark:bg-white/20">
        OR for ease, they are: {MY_SKILLS}
      </p>
    </CardWrapper>
  );
};

export default AboutMe;
