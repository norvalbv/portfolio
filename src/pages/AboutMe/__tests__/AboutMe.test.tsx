import React from 'react';
import renderer from 'react-test-renderer';

import AboutMe from '..';

describe('<AboutMe />', () => {
  test('should render correctly', () => {
    const tree = renderer.create(<AboutMe />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
