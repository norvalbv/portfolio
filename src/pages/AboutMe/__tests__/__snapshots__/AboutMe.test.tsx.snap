// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutMe /> should render correctly 1`] = `
<div
  className="relative mx-auto flex-1 grid w-8/12 place-items-center text-center"
>
  <div>
    <header
      className="flex items-center justify-between"
    >
      <section
        className="flex w-full flex-col gap-1"
      >
        <h1
          className="mx-auto mb-[4.5rem] w-max border-b pb-2 text-5xl font-semibold md:text-6xl"
        >
          H
          e
          l
          l
          o
           
          W
          o
          r
          l
          d
          !
        </h1>
      </section>
    </header>
    <p>
      Experienced Full Stack software engineer with 3 years of experience in web development. I am part of the core team that developed the entire 
      <a
        className="text-accent-secondary hover:underline"
        href="https://elysia.co"
        rel="noreferrer noopener"
        target="_blank"
      >
        Elysia
      </a>
       (a WAE subsidiary) cloud platform. I would love for you to check out my projects and even reach out!
    </p>
    <p
      className="my-10 text-accent-main"
    >
      My skills? They're floating around on the background. Have a look around :)
    </p>
    <p
      className="mx-auto w-6/12 min-w-[16rem] rounded border border-light-text bg-black/20 p-4 text-xs dark:border-dark-text dark:bg-white/20"
    >
      OR, they are: 
      JavaScript, TypeScript, React, Redux, Jest, Cypress, D3, Visx, HTML, CSS, TailwindCSS, Node, Express, AWS Serverless, CI/CD, Git, Bash, GitHub, Azure DevOps, Agile/Scrum/Kanban, SEO
    </p>
  </div>
</div>
`;
